velox/expression/Expr.cpp

```diff
@@ -999,9 +999,12 @@ Expr::PeelEncodingsResult Expr::peelEncodings(
   }
```

Comment:
NYC? shall we change DCHECK_EQ at l995 to VELOX_CHECK_EQ?

Reply:
I'm not sure I understand. Why is that check affected by this change?

Reply:
We can keep it separate. DCHECK_EQ will cause server crash in debug build. Not sure if we want to change it to VELOX_CHECK_EQ to throw a query